### ISSUE TITLE
Battery symbol [OR instead of AND] percentage 

### ISSUE THREAD

**Issue by bailarin2020** [iglance/iGlance#194]
**Is your feature request related to a problem? Please describe.**
It is redundant to have them both at the same time using up the precious menubar space

**Describe the solution you'd like**
To have the option to choose which one to have displayed (or both, you wish)

**Describe alternatives you've considered**
It is up to the dev

**Additional context**
Just to have the option to declutter the menu bar

**Comment by D0miH** [comment]
Hey @bailarin2020,
this is indeed redundant. Imo this is a good idea to let the user choose which combination he wants.

**Comment by bailarin2020** [comment]
Hi D0miH,
It would be great!
Looking forward for this update.
(So, I'm not closing this issue yet...right?)

Thanks.

**Comment by D0miH** [comment]
Correct. The issue will be closed when the feature is implemented and published :)
However I am not sure when this feature will be released.

**Comment by ck090** [comment]
Hi @D0miH, 
I can work on this issue and contribute to the development as asked by @bailarin2020.
I've gone through the contribution guidelines as well.
